packages/grafana-runtime/src/utils/DataSourceWithBackend.ts

```diff
@@ -45,6 +45,7 @@ export enum HealthStatus {
  * plugin.
  *
  * If the 'message' key exists, this will be displayed in the error message in DataSourceSettingsPage
+ * If the 'verboseMessage' key exists, this will be displayed in the expandable details in the error message in DataSourceSettingsPage
```

Comment:
can we also add a go struct to the SDK with this same shape?

Reply:
Do you mean replace the `[]byte` type for `JSONDetails` https://github.com/grafana/grafana-plugin-sdk-go/blob/master/backend/diagnostics.go#L63 with something like this

```
type CheckHealthResultsJSONDetails struct {
	Message string `json:"message,omitempty"`
	VerboseMessage   string `json:"verboseMessage,omitempty"`
}
```
so we are able to define the specific fields we are expecting and be more strict with the keys or do you mean something else?